docs/userguides/development.md

```diff
@@ -7,22 +7,73 @@ In this guide, we are going to show you more details on how to build an applicat
 You should have a python project with Silverback installed.
 You can install Silverback via `pip install silverback`
 
+## Project structure
+
+There are 3 suggested ways to structure your project:
+
+1. Create a `bot.py` file in the root directory of your project. This is the simplest way to define your bot project.
+
+2. Create a `bots/` folder in the root directory of your project. Then develop bots in this folder as separate scripts (Do not include a __init__.py file).
+
+3. Create a `bot/` folder with a `__init__.py` file that will include the instantiation of your `SilverbackApp()` object.
```

Comment:
suggestion(non-blocking): There's a lot of repetition in this file. Especially around these 3 points. Consider consolidating a bit.